### DISCUSSION TITLE
gui wont load after i enter mnemonics it tells me 'nonetype' object is not subscriptable

### DISCUSSION THREAD

**Discussion by tgore1** [Chia-Network/chia-blockchain#9140] [Support]
i was having issues with a  hdd it was saying they were invalid so i restarted computer and went to load gui and it came up asking for mnemonics which has never happened after entering them it popped up 'nonetype' object is not subscriptable but from log it appears to be farming i think but gui will not open

**Comment by tgore1** [comment]
invalid plots*

**Comment by sparkd82** [comment]
Hi, I have the same issue. Cannot start Chia. Any solution please?

**Comment by loppefaaret** [comment]
@tgore1 it sounds like you might have data consistency issues ? if you have failing drives - you might be loosing data ?

@sparkd82 you need to be more accurate at what you are doing, and what you are seeing in response - it is hard to identify any possibly issue from what you are explaining here, the possibilities are far too many to offer any solution

**Comment by Ron-ski** [comment]
I'm having the same issue, but with Flora. I killed the processes instead of Flax which was having problems. On restarting Flora it asked me to enter the 24 word mnemonics, then on pressing Saf to Skip I get the error  "'NoneType' object is not subscriptable" Can also confirm looking at the log it is farming. Once I press OK on the error message and then "Safe to Skip", I then get an error which just say "fingerprint", pressing OK on the error then allows you to press safe to skip again and we just get the finger print error.

I'm going to completely uninstall Flora and reinstall.

PS. I know my plots are valid as other coins are working fine.

Edit. Deleting keys, uninstalling, then deleting the Flora folders that are left behind and reinstalling with a DB backup cured it.

**Comment by OldButtKicking** [comment]
I found a work around for this on another coin. First create a new wallet, you don't need to keep track of the keys because you are just going to delete it. Now try and create your old wallet with the mnemonics . This worked for me then I just deleted the new wallet. I was installing a clean install from exe install on a new machine with windows 11.
